.github/workflows/build-master.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  push:
+    branches:
+      - master
+      # gh-mster-3 must be removed before merging this PR. It was added to test the pipeline.
```

Comment:
I have added a temporary commit to run the pipeline directly in this PR branch to have a better feedback loop.

Reply:
Removed the commit for final review.